eqc/mstore_eqc.erl

```diff
@@ -133,9 +131,9 @@ prop_gb_comp() ->
                     begin
                         os:cmd("rm -r " ++ ?DIR),
                         {S, T} = eval(D),
-                        List = ?G:to_list(T),
-                        List1 = [{?S:get(S, ?M, Time, 1), V} || {Time, V} <- List],
-                        ?S:delete(S),
+                        List = gb_trees:to_list(T),
+                        List1 = [{mstore:get(S, ?M, Time, 1), V} || {Time, V} <- List],
```

Comment:
Line 135 is too long:                         List1 = [{mstore:get(S, ?M, Time, 1), V} || {Time, V} <- List],.